package-lock.json

```diff
@@ -6627,6 +6627,74 @@
 			"resolved": "https://registry.npmjs.org/@leichtgewicht/ip-codec/-/ip-codec-2.0.4.tgz",
 			"integrity": "sha512-Hcv+nVC0kZnQ3tD9GVu5xSMR4VVYOteQIr/hwFPVEvPdlXqgGEuRjiheChHgdM+JyqdgNcmzZOX/tnl0JOiI7A=="
 		},
+		"node_modules/@libsql/client": {
+			"version": "0.1.1",
+			"resolved": "https://registry.npmjs.org/@libsql/client/-/client-0.1.1.tgz",
+			"integrity": "sha512-jzz0SVsceURufBfNtudvqOBtglUEyJzjvZn+QVwax/rhjh2rFjDy76pyJcNmYQQyZlUeyMlbwpNhvtOHD5Ut3w==",
+			"dependencies": {
+				"@libsql/hrana-client": "^0.3.3",
+				"@libsql/isomorphic-fetch": "^0.1.1",
+				"better-sqlite3": "^8.0.1",
+				"js-base64": "^3.7.5"
+			}
+		},
```

Comment:
Where does this come from?

Reply:
I reinstalled, likely something to do with D1